### ISSUE TITLE
[BUG] RectangleLabels fail after rotating the image

### ISSUE THREAD

**Issue by thesby** [heartexlabs/label-studio#641]
version: 0.9.1.post1
browser: chrome 88.0.4324.146
OS: MacOS Mojave 10.14.5

RectangleLabels cannot draw rectangle on right position after rotating the image.

**Comment by makseq** [comment]
Could you provide more details? I've checked the rotation [here](https://labelstud.io/playground/?config=%3CView%3E%3Cbr%3E%20%20%3CImage%20name%3D%22image%22%20value%3D%22%24image%22%2F%3E%3Cbr%3E%20%20%3CRectangleLabels%20name%3D%22label%22%20toName%3D%22image%22%3E%3Cbr%3E%20%20%20%20%3CLabel%20value%3D%22Airplane%22%20background%3D%22green%22%2F%3E%3Cbr%3E%20%20%20%20%3CLabel%20value%3D%22Car%22%20background%3D%22blue%22%2F%3E%3Cbr%3E%20%20%3C%2FRectangleLabels%3E%3Cbr%3E%3C%2FView%3E%3Cbr%3E) and it works well.

**Comment by thesby** [comment]
Image classification is the base template of my project. After rotating the image, I cannot draw rectangle on some positions. 
```html
<View style="display: flex;">
    <View style="width: 400px; padding-left: 1em; margin-right: 1em; background: #f1f1f1; border-radius: 3px;">
        <View style="box-shadow: 2px 2px 5px #999;
                    padding: 5px; margin: 5px;
                    border-radius: 5px;">
            <Header value="classes"/>
            <Choices name="choice" toName="image" showInLine="true" required="true">
                <Choice value="class1" background="blue"/>
                <Choice value="class2" background="green" />
                <Choice value="class3" background="blue"/>
                <Choice value="class4" background="green" />
            </Choices>
        </View>

        <View style="box-shadow: 2px 2px 5px #999;
                    padding: 5px; margin: 5px;
                    border-radius: 5px;">
            <Header value="categories"/>
            <Choices name="reason_choice" toName="image" showInLine="false">
                <Choice value="no problem" background="blue"/>
                <Choice value="others" background="blue"/>
            </Choices>
        </View>
        <RectangleLabels name="label" toName="image">
            <Label value="Area" background="green"/>
        </RectangleLabels>
    </View>

    <View>
        <Image name="image" value="$image_url" maxWidth="1440px"
                zoom="true" zoomControl="true" rotateControl="true"/>
    </View>
</View>
```

**Comment by thesby** [comment]
Another problem is that the rotated image will disappear if I use `display: flex`

**Comment by Dorokah** [comment]
I'm experiencing this problem too.

**Comment by xuchenCN** [comment]
Here is a problem while using rotationControl

```
<View>
  <Image name="image" value="$image" rotateControl="true"/>
  <RectangleLabels name="label" toName="image">
    <Label value="Airplane" background="green"/>
    <Label value="Car" background="blue"/>
  </RectangleLabels>
</View>
```

Draw a rectangle
![image](https://user-images.githubusercontent.com/2757089/119474823-2558b980-bd7f-11eb-9d72-7883265f9dae.png)

After rotation
![image](https://user-images.githubusercontent.com/2757089/119474934-3dc8d400-bd7f-11eb-9467-dcf6a1c5b39d.png)

**Comment by nicholasrq** [comment]
hey @xuchenCN 

what LS version you're using? we fixed this issue in 1.0.2 recently

**Comment by xuchenCN** [comment]
> hey @xuchenCN
> 
> what LS version you're using? we fixed this issue in 1.0.2 recently

Hi @nicholasrq 
I'm using label-studio-frontend v1.0.1 (Latest)

**Comment by xuchenCN** [comment]
Thansk @nicholasrq 
Build LSF from master branch, image and bbox are rotating expected.
But the image zoom without height limition it seems a bit weird.

**Comment by makseq** [comment]
This bug was fixed in [release/1.1.0](https://github.com/heartexlabs/label-studio/tree/release/1.1.0). Feel free to reopen if you have this problem again.
